LoopKitUI/Extensions/Environment+Colors.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Environment+Colors.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-07-29.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+private struct GuardrailColorsKey: EnvironmentKey {
+    static var defaultValue: GuardrailColors = GuardrailColors()
+}
+
+public extension EnvironmentValues {
+    var guardrailColors: GuardrailColors {
+        get { self[GuardrailColorsKey.self] }
+        set { self[GuardrailColorsKey.self] = newValue }
+    }
+}
+
+private struct CarbTintColorKey: EnvironmentKey {
+    static var defaultValue: Color = .green
```

Comment:
Do not make configurable items public global vars.

Reply:
hm... here it is conforming to the protocol. I don't think there is an alternative. What do you suggest?